pkg/network/tracer/offsetguess_test.go

```diff
@@ -158,56 +160,82 @@ func testOffsetGuess(t *testing.T) {
 		value := c.Value.(uint64)
 		switch c.Name {
 		case "offset_saddr":
+			log.Debug("offset_saddr: ", value)
```

Comment:
I think you want to use `t.Logf` here.

Reply:
You can also just add one log on line 325 instead of one for each of these cases here.